organization/ircam_oauth2_provider/provider.py

```diff
@@ -0,0 +1,42 @@
+from allauth.socialaccount import providers
+from allauth.socialaccount.providers.base import ProviderAccount
+from allauth.socialaccount.providers.oauth2.provider import OAuth2Provider
+
+
+class IrcamAuthAccount(ProviderAccount):
```

Comment:
I wonder how far the Ircam naming of the oauth2 tool and provider are required.
Because Mezzo should be generic as much as possible, all modules should be named with generic class titles. 
Could you please explain what are the _real_ Ircam specificity of those oauth classes?

Reply:
Oauth2 protocol let you define which properties are served by the auth server. So, the class IrcamAuthAccount is specific to our ircamauth server